### ISSUE TITLE
file closing 

### ISSUE THREAD

**Issue by KanduriC** [bionumpy/bionumpy#13]
* bionumpy version: 0.2.1
* Python version: 3.8
* Operating System: Mac

### Description

The following pythonic idiom is raising the following error.

### What I Did

```
import bionumpy as bnp
with bnp.open(some_file) as file:
    ......
```

<img width="841" alt="Screenshot 2022-10-15 at 07 53 14" src="https://user-images.githubusercontent.com/8395053/195985583-4d7a370b-c5e2-4ecb-8b22-2b072eea035d.png">

**Comment by knutdrand** [comment]
Have you fixed this in 'dev' @pavlovicmilena ?

**Comment by pavlovicmilena** [comment]
No, I fixed missing attributes in the __exit__(..) function, but I could also take a look at this?

**Comment by knutdrand** [comment]
That would be great!

**Comment by pavlovicmilena** [comment]
this should be fixed now on dev (commit f5044886730c17b6975ecae7c1d3cf0a6539014e)

**Comment by knutdrand** [comment]
Works for me now :)
